src/app/channels/channel-videos/channel-videos.component.html

```diff
@@ -1,9 +1,12 @@
 <div id="container">
-    <bs-video-list *ngIf="videos && videos.length > 0; else noVideos" [videos]="videos" [mode]="'grid'"></bs-video-list>
+    <bs-video-list *ngIf="videos && videos.length > 0 && !isLoading; else noVideos" [videos]="videos" [mode]="'grid'"></bs-video-list>
```

Comment:
Better idea would be to wrap both `bs-video-list` and `ng-template #noVideos` inside a `div` with `*ngIf="!isLoading"`.